ts/mixpanelConfig/mixpanelPropertyUtils.ts

```diff
@@ -101,3 +105,6 @@ export const welfareStatusHandler = (
   ) as Array<WalletCardBonus>;
   return idPayCards.map(card => card.name);
 };
+
+export const fontPreferenceHandler = (state: GlobalState): TypefaceChoice =>
```

Comment:
Do we need to wrap the selector's call around a function? Can't we just call it directly without this one?

Reply:
Addressed in https://github.com/pagopa/io-app/pull/6744/commits/36a950e86073ba9825ba76f043206bc518e207d6